website/docs/r/organizations_account.html.markdown

```diff
@@ -32,6 +30,7 @@ The following arguments are supported:
 * `iam_user_access_to_billing` - (Optional) If set to `ALLOW`, the new account enables IAM users to access account billing information if they have the required permissions. If set to `DENY`, then only the root user of the new account can access account billing information.
 * `parent_id` - (Optional) Parent Organizational Unit ID or Root ID for the account. Defaults to the Organization default Root ID. A configuration must be present for this argument to perform drift detection.
 * `role_name` - (Optional) The name of an IAM role that Organizations automatically preconfigures in the new member account. This role trusts the master account, allowing users in the master account to assume the role, as permitted by the master account administrator. The role has administrator permissions in the new member account. The Organizations API provides no method for reading this information after account creation, so Terraform cannot perform drift detection on its value and will always show a difference for a configured value after import unless [`ignore_changes`](https://www.terraform.io/docs/configuration/meta-arguments/lifecycle.html#ignore_changes) is used.
+* `close_account_on_deletion` - (Optional) If true, a deletion event will close the account. Otherwise, it will only remove from the organization
```

Comment:
Punctuation nit
```suggestion
* `close_account_on_deletion` - (Optional) If true, a deletion event will close the account. Otherwise, it will only remove from the organization.
```